ui/ui-enterprise/pages/destination/[id]/edit.tsx

```diff
@@ -155,6 +155,26 @@ export default function Page(props) {
                 </Form.Control.Feedback>
               </Form.Group>
 
+              <Form.Group controlId="syncMode">
+                <Form.Label>Sync Mode</Form.Label>
+                <Form.Control
+                  as="select"
+                  required={true}
```

Comment:
Since it's required in the form, that's a good hint that it's required in the action